crates/starknet/src/utils.rs

```diff
@@ -49,17 +49,23 @@ pub(crate) mod test_utils {
     use starknet_types::traits::HashProducer;
 
     use crate::constants::{
-        DEVNET_DEFAULT_CHAIN_ID, DEVNET_DEFAULT_GAS_PRICE, DEVNET_DEFAULT_HOST,
-        DEVNET_DEFAULT_INITIAL_BALANCE, DEVNET_DEFAULT_PORT, DEVNET_DEFAULT_TEST_SEED,
-        DEVNET_DEFAULT_TIMEOUT, DEVNET_DEFAULT_TOTAL_ACCOUNTS,
+        CAIRO_0_ACCOUNT_CONTRACT_PATH, DEVNET_DEFAULT_CHAIN_ID, DEVNET_DEFAULT_GAS_PRICE,
+        DEVNET_DEFAULT_HOST, DEVNET_DEFAULT_INITIAL_BALANCE, DEVNET_DEFAULT_PORT,
+        DEVNET_DEFAULT_TEST_SEED, DEVNET_DEFAULT_TIMEOUT, DEVNET_DEFAULT_TOTAL_ACCOUNTS,
     };
     use crate::starknet::starknet_config::StarknetConfig;
     use crate::utils::exported_test_utils::dummy_cairo_0_contract_class;
 
     pub fn starknet_config_for_test() -> StarknetConfig {
+        let account_contract_class =
+            Cairo0Json::raw_json_from_path(CAIRO_0_ACCOUNT_CONTRACT_PATH).unwrap();
         StarknetConfig {
             seed: DEVNET_DEFAULT_TEST_SEED,
             total_accounts: DEVNET_DEFAULT_TOTAL_ACCOUNTS,
+            account_contract_class: ContractClass::Cairo0(Cairo0ContractClass::RawJson(
+                account_contract_class.clone(),
```

Comment:
Just a small non-important fix, if you swap, the account_contract_class and account_contract_class_hash properties assignment, you can avoid cloning the contract_class.

Reply:
Nice tip. Swapped in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196/commits/66cdb45687f026939f01c4ccca766e28be31c329